website/content/en/docs/overview/operator-capabilities.md

```diff
@@ -182,6 +183,12 @@ The RED Method defines the three key metrics for every service in your architect
 * Errors (the number of those requests that are failing)
 * Duration (the amount of time those requests take)
 
+Note that by building projects using Operator-SDK or [Kubebuilder][kubebuilder] CLI tools your solutions
+leverage on [controller-runtime][controller-runtime] which provides the above metrics exported by default. 
```

Comment:
@camilamacedo86 Did you mean to refer here to the metrics documentation?

Reply:
I think that the link yo the default metrics should be here, since you mention them below,not above